src/Contrib/Dispatcher/IntegrationEvents/Masa.Contrib.Dispatcher.IntegrationEvents/Options/IntegrationEventOptions.cs

```diff
@@ -199,6 +199,24 @@ public int DeleteBatchCount
 
     public Func<DateTime>? GetCurrentTime { get; set; }
 
+    private int _executeInterval = 1;
```

Comment:
Line 22,39,57,74,93. Remove this argument from the method call; it hides the caller information.

Reply:
Processed